modules/UI/util/UIUtil.js

```diff
@@ -447,70 +447,15 @@ const IndicatorFontSizes = {
     },
 
     /**
-     * Gets an "indicator" span for a video thumbnail.
-     * If element doesn't exist then creates it and appends
-     * video span container.
-     *
-     * @param {object} opts
-     * @param opts.indicatorId {String} - identificator of indicator
-     * @param opts.videoSpanId {String} - identificator of video span
-     * @param opts.content {String} HTML content of indicator
-     * @param opts.tooltip {String} - tooltip key for translation
-     *
-     * @returns {HTMLSpanElement} indicatorSpan
-     */
-    getVideoThumbnailIndicatorSpan(opts = {}) {
```

Comment:
This is a get or create method for the indicators, which is unnecessary now because the indicators are declared with jsx.